iguana/assets.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -eo pipefail
+
+source pubkey.txt
+# source $HOME/config.nn
+
+seed_ip=`getent hosts zero.kolo.supernet.org | awk '{ print $1 }'`
+komodo_binary="$HOME/komodo/src/komodod"
+cli_binary="$HOME/komodo/src/komodo-cli"
+delay=20
```

Comment:
```suggestion
delay=${delay:-20}
```

Reply:
Thanks for hint. It's moved to config in newer version